app/src/main/java/tech/zafrani/companionforpubg/utils/PUBGNewsFetch.java

```diff
@@ -0,0 +1,65 @@
+package tech.zafrani.companionforpubg.utils;
+
+import android.os.AsyncTask;
+import android.support.annotation.NonNull;
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+
+import java.io.IOException;
+
+import tech.zafrani.companionforpubg.adapters.NewsAdapter;
+import tech.zafrani.companionforpubg.models.NewsItem;
+
+public class PUBGNewsFetch extends AsyncTask<Void, NewsItem, Void>  {
+    private final NewsAdapter adapter;
+
+    public PUBGNewsFetch(@NonNull final NewsAdapter adapter) {
+        this.adapter= adapter;
+    }
+
+    @Override
+    protected Void doInBackground(final Void... voids) {
+
+        final Document document;
+        try {
+            document = Jsoup.connect("https://www.playbattlegrounds.com/news.pu").get();
+            for (final Element row : document.select("ul#allList li")) {
+
+                final String title = row.select(".title").text();
```

Comment:
I would store all of these strings like .title as private static final strings of this class. Each for loop will create a new instance of the same value.